### ISSUE TITLE
Property grid imports `@itwin/presentation-common` as a dev dependency but doesn't treat it as such

### ISSUE THREAD

**Issue by johnnyd710** [iTwin/viewer-components-react#1006]
In `packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx`, [`Key` is imported from `@itwin/presentation-common`
](https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx#L14) but `@itwin/presentation-common` is [listed as a devDependency](https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/property-grid/package.json#L87). 

```js
import { Key } from "@itwin/presentation-common";
```

```json5
{
 "devDependencies": {
    // ...
    "@itwin/presentation-common": "^4.4.0",
    // ...
   },
}
```

Not sure if this is supposed to be a type import - otherwise `@itwin/presentation-common` should be listed as a dependency or peerDep

**Comment by aruniverse** [comment]
Looks like not used just for types, https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx#L89

**Comment by saskliutas** [comment]
@johnnyd710 Published `@itwin/property-grid-react@1.11.0` with fixed dependencies
